src/test/groovy/pl/exsio/querydsl/entityql/groovy/native_sql/it/BaseClassForTests.groovy

```diff
@@ -0,0 +1,46 @@
+package pl.exsio.querydsl.entityql.groovy.native_sql.it
+
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.Column
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.Table
+import pl.exsio.querydsl.entityql.groovy.config.enums.by_name.UserTypeByName
+import pl.exsio.querydsl.entityql.groovy.config.enums.by_ordinal.UserTypeByOrdinal
+import spock.lang.Specification
+
+abstract class BaseClassForTests extends Specification {
```

Comment:
please rename to DynamicTestsBase

Reply:
correction, rename to RuntimeFixedTestBase

Reply:
renamed to RuntimeFixedTestBase